### ISSUE TITLE
Resolver exists twice with Browserify

### ISSUE THREAD

**Issue by ivan-kleshnin** [ericclemmons/react-resolver#27]
Everything looks fine to me

```javascript
// app.js
import {Resolver} from "react-resolver";

window.router = ReactRouter.create({
  routes: routes,
  location: HistoryLocation
});

window.router.run((Handler, state) => {
  Resolver.render(<Handler/>, document.getElementById("main")));
});

// home.js
class Home extends React.Component {
  static displayName = "Home"

  render() {
    return <div>Home</div>;
  }
}

export default Resolver.createContainer(Home, {
  resolve: {}
});
```

Getting **Resolver is not defined in either `context` or `props`.  (Perhaps missing a root <Container resolver={new Resolver()} />?)**

React: 0.13.1
React-Resolver: 0.1.1

**Comment by ericclemmons** [comment]
Wow, that's interesting!  `Resolver.render` [wraps your `<Handler />` in a `<Container />`][1] to pass down context, so everything you have looks just like the [example][2].

Can you confirm that in your console (Chrome/Firefox) you're not seeing this message more than once?
> Download the React DevTools for a better development experience: http://fb.me/react-devtools

[1]: https://github.com/ericclemmons/react-resolver/blob/db21b221135e7cc7db40ab333f6e0c559e986036/src/Resolver.js#L153-L161
[2]: https://github.com/ericclemmons/react-resolver/blob/db21b221135e7cc7db40ab333f6e0c559e986036/examples/stargazers/main.js#L13-L15

**Comment by srph** [comment]
This seems to happen to me too.

```
Module not found: Cannot resolve 'file' or 'directory' ./Resolver in <path/to/dist>
```

**Comment by ericclemmons** [comment]
@ivan-kleshnin I just released `v1.1.3` with a fix from #28.

Can you confirm this bug still exists?

**Comment by ivan-kleshnin** [comment]
I think you imply second React instance :wink:
Yep, that's look very like that. Browserify has hard times with both `react` and `react/addons` used.
I always use pure `react` and you import 
```javascript
var _React = require("react/addons");
```
in react-resolver so I end with two instances of them. What do you propose to do then?

**Comment by ericclemmons** [comment]
Man, this sucks.  I had to switch from Browserify to Webpack because of this issue (not just this project, but Flummox & others have it as well) just to get tests working.

I tried to use `React.CloneElement` instead of `React.addons.cloneWithProps`, but it doesn't maintain context as it should.

I think we're waiting for another React release until this is resolved.  

Maybe I can just reference this?
> https://github.com/facebook/react/blob/master/src/utils/cloneWithProps.js

**Comment by srph** [comment]
Won't `import cloneWithProps from 'react/lib/cloneWithProps` work? Probably not

**Comment by ericclemmons** [comment]
Maybe? I'll try!

**Comment by ivan-kleshnin** [comment]
> Man, this sucks. I had to switch from Browserify to Webpack because of this issue (not just this project, but Flummox & others have it as well) just to get tests working.

Subscribe. I'm already in the transition process. :smiley: So webpack should handle case, right?

**Comment by ericclemmons** [comment]
This should be fixed via `v1.1.4` (#32)!
